src/Allegro.CosmosDb.Migrator/README.md

```diff
@@ -0,0 +1,21 @@
+# Introduction 
+This project contains fully working service that enables real time as is migration between two CosmosDb containers.
+
+It contains:
+- migration api
+- migration statistics api
+
+# Getting Started
```

Comment:
Maybe we should briefly describe here how to use it (copy the solution locally, set connection strings, run with script etc)

Reply:
added some simple instruction